src/app/Layouts/defaultPageWrapper.jsx

```diff
@@ -22,27 +72,24 @@ const PageWrapper = ({
   <DialContextProvider dials={dials}>
     <ServiceContextProvider service={service}>
       <GlobalStyle />
-      <RequestContextProvider
-        bbcOrigin={bbcOrigin}
+      <PageWithRequestContext
+        data={data}
         id={id}
         isAmp={isAmp}
+        bbcOrigin={bbcOrigin}
         pageType={pageType}
         service={service}
       >
-        <ServiceWorkerContainer />
-        <ManifestContainer />
-        <ConsentBanner />
-        <HeaderContainer />
         {children}
-        <FooterContainer />
-      </RequestContextProvider>
+      </PageWithRequestContext>
     </ServiceContextProvider>
   </DialContextProvider>
 );
 
 PageWrapper.propTypes = {
   bbcOrigin: string,
   children: node.isRequired,
+  data: shape({ any }).isRequired,
```

Comment:
Can it really be `any` shape? or do we have a set structure for what the data should look like?